File: GO/euler010.go
package main

import (
	"fmt"
	"math"
)

func main() {
	fmt.Println(solve())
}

func solve() int {
	LIMIT := 2000000
	res := 0

	for i := 2; i < LIMIT; i++ {
		if is_prime(i) {
			res += i
		}
	}
	return res
}

func is_prime(p int) bool {
	for i := 2; i <= int(math.Sqrt(float64(p))); i++ {
		if p%i == 0 {
			return false
		}
	}
	return true
}

File: C#/euler010.cs
using System;

namespace ProjectEuler {
    class euler010 {
        static void Main() {
            Console.WriteLine(Solve());
        }

        static Int64 Solve() {
            Int64 res = 0;
            for (Int64 i = 2; i < 2000000; i++) {
                if (is_prime(i)) {
                    res += i;
                }
            }
            return res;
        }

        static bool is_prime(Int64 n) {
            long end = Convert.ToInt64(Math.Sqrt(n));
            for (Int64 i = 2; i <= end; i++) {
                if (n%i == 0) {
                    return false;
                }
            }
            return true;
        }
    }
}